content/en/docs/collector/architecture.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Architecture
+weight: 28
+# prettier-ignore
+cSpell:ignore: # TODO: Add keywords to ignore.
```

Comment:
```suggestion
cSpell:ignore: fanoutconsumer probabilisticsampler spanmetrics zpages
```

this will satisfy the cspell checks

Reply:
Thanks. Is this something I could have tested on my own or is that part of CI only available to maintainers?

Reply:
yes, with `npm run check:spelling`